packages/useWorker/src/lib/createWorker.ts

```diff
@@ -0,0 +1,53 @@
+import { createContext, dataPlaceholder } from "isoworker";
```

Comment:
@101arrowz
what is `dataPlaceholder` ? 🤔 
I saw that "isoworker" has no exported member 'dataPlaceholder'